internal/application/origin_test.go

```diff
@@ -0,0 +1,78 @@
+package application
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/epinio/epinio/pkg/api/core/v1/models"
+)
+
+func Test_buildBodyPatch(t *testing.T) {
```

Comment:
I'm not sure our make target will run non-ginkgo tests: https://github.com/epinio/epinio/blob/main/Makefile#L71-L72

Also, although this is just a unit test, I would still stick to ginkgo syntax for consistency. E.g. like this one: https://github.com/epinio/epinio/blob/main/internal/s3manager/s3manager_test.go

I find the tests descriptions helpful when trying to understand what is being tested.